### PR TITLE
Updated a translate from English to Russian

### PR DESCRIPTION
**Pull request by NikLoginov**

### CODE REVIEW

File: docs/ru/operations/clickhouse-keeper.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
----
+[---
```
Comment (alexey-milovidov) [inline docs/ru/operations/clickhouse-keeper.md:1]: That's strange.

**Review by alexey-milovidov — CHANGES_REQUESTED** [review]
.

File: docs/ru/operations/clickhouse-keeper.md (hunk 1 above)
Comment (NikLoginov) [reply docs/ru/operations/clickhouse-keeper.md:1]: @alexey-milovidov What do you mean? Russian article very old (

**Review by den-crane — APPROVED** [review]

File: docs/ru/operations/clickhouse-keeper.md (hunk 1 above)
Comment (alexey-milovidov) [reply docs/ru/operations/clickhouse-keeper.md:1]: You replaced the whole article with the link to your personal profile.

File: docs/ru/operations/clickhouse-keeper.md [hunk 2]
```diff
@@ -343,3 +343,4 @@ clickhouse-keeper-converter --zookeeper-logs-dir /var/lib/zookeeper/version-2 --
 5. Запускайте остальные ноды кластера по одной и проверяйте, что команда `mntr` возвращает `follower` в выводе состояния `zk_server_state` перед тем, как запустить следующую ноду.
 6. Пока нода работает в режиме восстановления, лидер будет возвращать ошибку на запрос `mntr` пока кворум не будет достигнут с помощью новых нод. Любые запросы от клиентов и последователей будут возвращать ошибку.
 7. После достижения кворума лидер перейдет в нормальный режим работы и станет обрабатывать все запросы через Raft. Удостоверьтесь, что запрос `mntr` возвращает `leader` в выводе состояния `zk_server_state`.
+](https://github.com/NikLoginov/ClickHouse/)
```
Comment (alexey-milovidov) [inline docs/ru/operations/clickhouse-keeper.md:346]: ^ This line.

File: docs/ru/engines/table-engines/mergetree-family/replication.md [hunk 3]
```diff
@@ -6,6 +6,24 @@ sidebar_label: "Репликация данных"
...
+    '/clickhouse/tables/{shard}/table_name',
+    '{replica}',
+    ver
+)
+```
+
+на:
+
+```sql
+ENGINE = ReplicatedReplacingMergeTree
```
Comment (alexey-milovidov) [inline docs/ru/engines/table-engines/mergetree-family/replication.md:23]: But `ver` is still needed.

**Comment by robot-ch-test-poll1** [comment]
This is an automated comment for commit 04881c3869b14362600a6938fda3e0b8307e9866 with description of existing statuses. It's updated for the latest CI running
The full report is available [here](https://s3.amazonaws.com/clickhouse-test-reports/50443/04881c3869b14362600a6938fda3e0b8307e9866/ci_running.html)
The overall status of the commit is 🔴 failure

<table><thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody><tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td>🔴 failure</td></tr>
</table>

**Comment by alexey-milovidov** [comment]
Changes needed.

**Pull request closed without merging**